haskell-src/Concordium/Types/ProtocolVersion.hs

```diff
@@ -174,6 +174,10 @@ module Concordium.Types.ProtocolVersion (
     AVSupportsFlexibleCooldown,
     PVSupportsFlexibleCooldown,
 
+    -- * Account suspension support
+    supportsAccountSuspension,
```

Comment:
Can we call this something different like `supportsValidatorSuspension`? I think "Account suspension" suggests a more general freezing of the account.

Reply:
renamed to `supportsValidatorSuspension`.